### PR TITLE
FIX: Center phase maps around central mode, avoiding FoV-related outliers

### PR DESCRIPTION
**Pull request by effigies**
First pass at https://github.com/poldracklab/sdcflows/issues/83#issuecomment-586391481.

Will update to follow https://github.com/poldracklab/sdcflows/issues/83#issuecomment-586411666, shortly.

cc @oesteban @ins0mniac2 @dorianps

### CODE REVIEW

File: sdcflows/interfaces/fmap.py [hunk 1]
```diff
@@ -661,19 +661,31 @@ def _delta_te(in_values, te1=None, te2=None):
...
     data = im.get_fdata(dtype='float32')
     hdr = im.header.copy()
 
-    # First center data around 0.0.
-    data -= mode(data, axis=None)[0][0]
+    dmin, dmax = data.min(), data.max()
+
+    # Find the mode ignoring outliers on the far max/min, to allow for junk outside the FoV
+    fudge = 0.05 * (dmax - dmin)
+    mode = stats.mode(data[(data > dmin + fudge) & (data < dmax - fudge)])[0][0]
```
Comment (mgxd) [inline sdcflows/interfaces/fmap.py:673]: We still want to compute over the whole array right?

```suggestion
    mode = stats.mode(data[(data > dmin + fudge) & (data < dmax - fudge)], axis=None)[0][0]
```

Comment (effigies) [reply sdcflows/interfaces/fmap.py:673]: Boolean indexing flattens the array.

**Review by mgxd — APPROVED** [review]
Seems much more robust

**Review by oesteban — APPROVED** [review]

**Comment by effigies** [comment]
Accidentally merged the CI fixes into `master` instead of `maint/1.0.x`, so now they're in here, too.

**Pull request merged**